lib/config/decrypt.spec.ts

```diff
@@ -208,5 +216,92 @@ describe('config/decrypt', () => {
         CONFIG_VALIDATION,
       );
     });
+
+    it('handles EC/AES-GCM encryption', async () => {
+      GlobalConfig.set({ privateKey: toBase64(privateEc) });
+      config.encrypted = {
+        token:
+          'eyJrIjp7Imt0eSI6IkVDIiwieCI6IlM0TFhnVUJHMUNlSXVUU1ZiZEg3MHZpSjV6dUxaRFNydFJ2ZUhVTHBBYnBMTXAwQ0tyeVozWm5RN1lKQjh3N2MiLCJ5IjoiZVpQNFZzT1hXZGxRYkNWYWxSakVWNEFzTFBxYmpKdW1mR1dfTzlZbG9HM2dqbzd5enJfUXQtbXl2SG5LdkZheiIsImNydiI6IlAtMzg0In0sImkiOiJNRDVMb2hUMmh2VS9SYXA3IiwibSI6IkptMERpUmNtbEFEU2JjeVdnWmJlUXl4QkdUbU55aVlkV2NDa0tyVnhQZStTdEdMMlBwWT0ifQ==',
+      };
+
+      const res = await decryptConfig(config, 'some/def');
+      expect(res.encrypted).toBeUndefined();
+      expect(res.token).toBe('123');
+      await expect(decryptConfig(config, 'abc/defg')).rejects.toThrow(
+        CONFIG_VALIDATION,
+      );
+    });
+  });
+
+  describe('tryDecryptEc', () => {
+    it('decrypts', async () => {
+      // generate a new key pair in browser for each encryption
+      // private key will be thrown away after encryption,
+      // so only renovate can decrypt with private key
```

Comment:
```suggestion
      // Generate a new key pair in the browser for each encryption,
      // this private key will be thrown away after encryption,
      // so that only Renovate can decrypt with the private key.
```